### PR TITLE
Av user profile

### PR DESCRIPTION
**Pull request by AnnaVi11arrea1**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (marywebby) [inline app/controllers/comments_controller.rb:26]: good job implementing `redirect_back`

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -8,6 +8,10 @@ def index
     render "users/index"
   end
 
+  def show
+    @user = User.find_by(username: params.fetch(:username))
```
Comment (marywebby) [inline app/controllers/users_controller.rb]: 👍

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+          <li class="list-group-item">
+            <div class="d-flex">
+              <div class="flex-shrink-0">
+                <img src="..." alt="...">
+              </div>
+              <div class="flex-grow-1 ms-3">
+                <h5 class="mt-0"><%= comment.author.username %></h5>
```
Comment (marywebby) [inline app/views/users/show.html.erb:22]: nice work!

File: config/routes.rb [hunk 4]
```diff
@@ -8,4 +8,7 @@
   resources :likes
   resources :photos
 
+
```
Comment (marywebby) [inline config/routes.rb]: Make sure you get rid of the duplicate `devise_for :users`, it is in this file twice!

**Review by marywebby — APPROVED** [review]
Make sure you remove the duplicated code, but otherwise great work!

**Review by dgarcia13-99 — APPROVED** [review]
Great job making edits! Your show.html.erb file looks especially well formatted!

**Pull request merged**